content/actions/sessions.md

```diff
@@ -61,3 +61,6 @@ module Web
   end
 end
 ```
+### Sharing session values between applications
+
+To share session values, defined in one application, we must provide the same session secret to all the applications where we need those values.
```

Comment:
Would it be worth duplicating the example code block above to this section, and then displaying two application classes with the same session key set for both?

Reply:
Thank you for the suggestion, I've made the changes.